src/DecisionTree.jl

```diff
@@ -83,6 +83,31 @@ function print_tree(leaf::Leaf, depth=-1, indent=0; feature_names=nothing)
     println("$(leaf.majority) : $(ratio)")
 end
 
+"""
+       print_tree(tree::Node, depth=-1, indent=0; feature_names=nothing)
+
+Print a textual visualization of the given decision tree `tree`.
+In the example output below, the top node look at if 
```

Comment:
Minor grammatical suggestion:

```suggestion
In the example output below, the top node considers whether 
```